_data/people/cvarni.yml

```diff
@@ -29,3 +29,10 @@ presentations:
   meetingurl: https://indico.cern.ch/event/1044816/
   project: acts
   location: CERN
+- title: ATLAS Track Reconstruction for HL-LHC using ACTS software
+  date: 2021-07-20
+  url: https://indico.cern.ch/event/1061445/contributions/4460829/attachments/2284847/3883288/TrackRecoATLAS.pdf
```

Comment:
The PDF links can change, it's better to link to the contribution (which does not change)

Reply:
Hi @henryiii , I've replaced the url with the contribution link